db/migrate/20200520210200_update_chargeback_reports_startpage.rb

```diff
@@ -0,0 +1,26 @@
+class UpdateChargebackReportsStartpage < ActiveRecord::Migration[5.2]
+  class User < ActiveRecord::Base
+    serialize :settings, Hash
+    include ActiveRecord::IdRegions
+  end
+
+  def up
+    say_with_time 'Updating starting page for users who had chargeback reports set' do
+      User.in_my_region.select(:id, :settings).each do |user|
```

Comment:
Why just in your region? Maybe I'm wrong, but that's not all the users :confused:

Reply:
@skateman i made this change to be consistent with changes requested here https://github.com/ManageIQ/manageiq-schema/pull/486#pullrequestreview-416181237

Reply:
it's better to not touch records from another region in data migrations because it can potentially break replication (though usually it's ok)